### ISSUE TITLE
OpenHPC Leap 15.3 Problem: 'kmod = 25-6.7.1' needed by the to be installed kmod-compat-25-6.7.1.x86_64 

### ISSUE THREAD

**Issue by windsleigh** [openhpc/ohpc#1602]
There is an issue when running 
---
```
[sms]# wwmkchroot -v opensuse-15.3 $CHROOT # create base image
```
---
```
Problem: nothing provides 'kmod-compat-25-6.7.1.x84_64' needed by the to be installed kmod-compat-25-6.7.1.x84_64
Solution 1: do not install kmod-compat-25-6.7.1.x84_64
Solution 2: break kmod-compat-25-6.7.1.x84_64 by ignoring some of its dependencies```

It happens when building the initial BOS image from the 2.x docs from OpenSUSE leap 15.3 Install guide(with Warewulf + Slurm)

**Comment by koomie** [comment]
We noticed this recently as well in our CI - appears to be related to changes in upstream Leap.  We have applied a fix in the Warewulf `wwmkchroot` recipe that will be available in the next release.  It's a pretty simple fix if you want to do it locally.  Essentially, edit the following file:

`/usr/lib/warewulf/wwmkchroot/opensuse-15.3.tmpl`

and replace `kmod-compat` with `suse-module-tools`.  See 40bdba3 for example.
